lib/puppet/provider/group/windows_adsi.rb

```diff
@@ -33,6 +33,7 @@ def members_insync?(current, should)
     if @resource[:auth_membership]
       current_users == specified_users
     else
+      return true if specified_users.empty?
```

Comment:
could do that as well, just not a fan of branching in that way and if it ever gets reordered, don't want the meaning to be lost.